src/encoded/static/components/navigation/components/TestWarning.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
 
 import React from 'react';
+import { console, isSelectAction, memoizedUrlParse, analytics } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['memoizedUrlParse' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5001501)